ext/civiimport/Civi/BAO/Import.php

```diff
@@ -251,4 +251,32 @@ private static function getAllFields(string $tableName): array {
     return array_merge(self::getFieldsForTable($tableName), self::getSupportedFields());
   }
 
+  /**
+   * Defines the default key as 'id'.
+   *
+   * @return array
+   */
+  public function keys() {
+    static $keys;
+    if (!isset($keys)) {
+      $keys = ['_id'];
+    }
+    return $keys;
```

Comment:
why do we have all this rather than just `return ['_id']` ?

Reply:
ok updated to that @eileenmcnaughton